bundles/config.en_us.json

```diff
@@ -3,6 +3,7 @@
     "ripe_commons.holder.holder-second-label.label": "3D",
 
     "ripe_commons.modal.cancel": "Cancel",
+    "ripe_commons.modal.clear": "Clear Initials",

```

Comment:
Not needed, right?

Reply:
It is used for the text label of the button "Clear Initials"
`{{ "ripe_commons.modal.apply" | locale }}` in `white_personalization.vue`
https://github.com/ripe-tech/ripe-white/commit/38d77d1f46b7864adb70e37751c64d0dbd4d25ec#diff-92d633628a5f85739498a60630ee9158R14

I can make this hardcoded instead, should I?

Reply:
Make it hard-coded for now, we need to revisit localisation in locale. The reason why this can't be here is that this is specific to "ripe-commons-pluginus" strings, and this is specific to White. We must find a way to have a `bundle` on White but for now let's go with the simple approach.

tl,dr: do this on White instead.